### PR TITLE
update psr/http-message ^2.0 & add ext-mbstring & update stefanak-michal/bolt ^5.1 -> ^6.0

### PR DESCRIPTION
**Pull request by lcp0578**
update psr/http-message ^2.0

### CODE REVIEW

**Comment by lcp0578** [comment]
When I use it for symfony 6.4 it need psr/http-message ^2.0

File: composer.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
         "php": "^8.0",
         "laudis/typed-enum": "^1.3.2",
         "php-http/discovery": "^1.13",
-        "psr/http-message": "^1.0",
+        "psr/http-message": "^1.0|^2.0",
```
Comment (stefanak-michal) [inline composer.json:27]: Does it need to keep the old version?

Comment (transistive) [reply composer.json:27]: I'd say so yes to maintain backwards compatibility

Comment (stefanak-michal) [reply composer.json:27]: your project is php^8 and with the change 1.1 to 2.0 ( https://github.com/php-fig/http-message/compare/1.1...2.0 ) you have nothing to worry about. It's just added argument types and return types.

**Comment by lcp0578** [comment]
I'm not sureneed to add constraints, 

`"ext-mbstring": "*"`

although Blot already has constraints, 
https://github.com/neo4j-php/Bolt/blob/master/composer.json#L12
but when I require this  libraries, composer didn't tell me that I needed mbstring extensions, which really wasted a lot of my time, and I found out through debugging that I really didn't have it, but it's not so intuitive, and I hope it will reduce the time of other people who use this library.

**Comment by lcp0578** [comment]
I think 
`"php": "^8.0",`
https://github.com/neo4j-php/neo4j-php-client/blob/main/composer.json#L24
so,  
image

https://github.com/neo4j-php/Bolt/releases/tag/v6.0.0

I'd like PHP's version constraints to be consistent.

**Comment by stefanak-michal** [comment]
If you are updating more than psr/http-message you should update PR name.

**Comment by lcp0578** [comment]
> If you are updating more than psr/http-message you should update PR name.

I update the PR name, thx.

**Comment by lcp0578** [comment]
@transistive If you have time to deal with this PR, my team and I urgently need to update a version.

**Pull request merged**